basis/tools/continuations/continuations.factor

```diff
@@ -26,6 +26,47 @@ SYMBOL: break-hook
 
 \ break t "break?" set-word-prop
 
+SYMBOL: hit-counter
+
+: (hit-count-inc) ( -- )
+    hit-counter get dup
```

Comment:
Probably should just be ``hit-counter inc``

Reply:
Yes, better name. I wrote this a long time ago.

Reply:
Not just a name, you can use ``inc`` from namespaces